blockchain/indexers/txindex.go

```diff
@@ -81,17 +86,28 @@ var (
 //
 // The serialized format for the keys and values in the tx index bucket is:
 //
-//   <txhash> = <block id><start offset><tx length>
+//   <txhash> = <block id><start offset><tx length><block index>
```

Comment:
What will be the migration plans for the indexes? Always drop and recreate if enabled? Or alert and let the user do it?

Reply:
I decided to open another PR (#1520) that contains the migration bits.